gettsim/data/e_st_abzuege.yaml

```diff
@@ -1,56 +1,70 @@
-kifreib:
+kifreib_s_exm:
   name:
-    de: Kinderfreibetrag (sächliches Existenzminimum des Kindes sowie Betreuungs- und Erziehungs- oder Ausbildungsbedarf)
+    de: Kinderfreibetrag (sächliches Existenzminimum des Kindes).
     en:
   description:
     de: § 32(6) EStG
     en:
   values:
     1984-01-01:
-      value: 221
-      note:
+      value: 110
+      note: Wert in Euro. Der Betrag laut Gesetz ist 216 DM.
     1986-01-01:
-      value: 1270
-      note:
+      value: 635
+      note: Art. 1 G. v. 26.06.1985 BGBl. I S. 1153. Wert in Euro. Der Betrag im Gesetz lautet 1242 DM.
     1990-01-01:
-      value: 1546
-      note: .
-    1994-01-01:
-      value: 2098
-      note:
+      value: 773
+      note: Wert in Euro. Wert laut Gesetz 1512 DM.
+    1992-01-01:
+      value: 1049
+      note: Wert in Euro. Wert laut Gesetz 2102 DM.
```

Comment:
I do not find the 2102 DM value.

https://offenegesetze.de/veroeffentlichung/bgbl1/1992/9#page=8 shows a few changes:
* replace "1512" by "2052",
* replace "3024" by "4104".

Should the DM value be 2052?

If so, please also add the BGBl. page to the note key.

Reply:
No the DM values are correct. They refer to the couple though.